lib/backend/lite/lite.go

```diff
@@ -320,14 +317,12 @@ func (l *Backend) createSchema() error {
            kv_key TEXT NOT NULL,
            kv_modified INTEGER NOT NULL,
            kv_expires DATETIME,
-           kv_value BLOB
+           kv_value BLOB,
+           kv_revision TEXT
```

Comment:
```suggestion
           kv_revision TEXT NOT NULL
```

Reply:
I wasn't thinking about that, fair point.

(we could have the revision of the deleted item, it's just not useful)

Reply:
I don't really like the inconsistency between the freshly created table and the table with the added columns tho, can we change both this and `revision` in `kv` to `TEXT NOT NULL DEFAULT ""` to match the column after `migrateRevision`, or have an extra parameter for the column constraint in `migrateRevision`?